lib/ukrand/include/sys/random.h

```diff
@@ -37,6 +37,7 @@ extern "C" {
 #endif
 
 #include <sys/types.h>
+#include <uk/arch/random.h>
 
 #define GRND_NONBLOCK     0x01
 #define GRND_RANDOM       0x02
```

Comment:
Now that hardware randomness is available, should we also implement these flags as described in [man getrandom](https://man7.org/linux/man-pages/man2/getrandom.2.html)?

Reply:
I think my previous suggestion is not relevant as none of these flags makes a difference for our usecase:
- `GRND_RANDOM` is used to ensure that you draw entropy from `/dev/random` instead of `/dev/urandom` i.e. the random numbers you get are better suited for cryptographic operations.
In our case, you choose your unique entropy source (either chacha CSPRNG or hardware pass-through) using `make menuconfig`, and therefore `getandom` will return random bytes from the previously selected source.
- `GRND_NONBLOCK` is used to avoid blocking if not enough entropy is available (e.g. to avoid blocking at boot time). This does not apply to us either, as we currently draw entropy from the hardware (via RDRAND / RNDR) so there is no concern about blocking.